# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1360475963 18000
#      Sun Feb 10 00:59:23 2013 -0500
# Node ID 31a0cbabf8dc4ca56d3cd99f2068803e19df6a26
# Parent  23a7661e529ae9bfc91693618f8c314c31f695ca
Added tag ss-3-7-2 for changeset faefa1bea8dd

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -66,8 +66,9 @@ ba4d6343524b406b0d15aee34579f80783581c54
 704f7895eef03008dd79848eb9da4bfb40787d73 release-3-6-0
 f947d2922febf12dcd1fb6e21b356756ecb54e55 rc-3-6-2-0
 4460c4fb20e6a5d3b1972fa737d4e00eb921545a rc-3-6-2-2
 551566201318bf615b27c60ccf9368f4844008bd release-3-6-2
 a95432e7309ca6fc776c02939264bb6d443f3525 release-3-6-3
 2e8eb9ac43a5f8cfaf0423814a312ed47cb80485 rc-3-6-4-0
 df1aceb8f0bc6b5b5062907931cc663467f57d93 ss-3-7-1
 858cbf6fc2ec1c232f5cf1d75dc344439b39a89c rc-3-6-4-1
+faefa1bea8ddae3cab170afdeab68d3d15c4e623 ss-3-7-2
